### ISSUE TITLE
Icon injection: wrong place

### ISSUE THREAD

**Issue by dasistwas** [Wunderbyte-GmbH/moodle-local_edusupport#4]
The icon is injected 2 times:

![Screenshot_2021-03-03 GIZ UAT Administration Search](https://user-images.githubusercontent.com/1620975/109798177-b9046900-7c1a-11eb-9c66-12254725e9bb.png)

**Comment by dasistwas** [comment]
This is the admin menu bar:

```
<ul class="nav nav-tabs" role="tablist" id="yui_3_17_2_1_1614770436101_48">
<!-- First the top most node and immediate children -->
    <li class="nav-item">
        <a class="nav-link" href="#linkroot" data-toggle="tab" role="tab" aria-selected="false" tabindex="-1">Site administration</a>
    </li>
<!-- Now the first level children with sub nodes -->
                <li class="nav-item">
                    <a class="nav-link" href="#linkusers" data-toggle="tab" role="tab" aria-selected="false" tabindex="-1">Users</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="#linkcourses" data-toggle="tab" role="tab" aria-selected="false" tabindex="-1">Learning</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="#linkgrades" data-toggle="tab" role="tab" aria-selected="false" tabindex="-1">Grades</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="#linkmodules" data-toggle="tab" role="tab" aria-selected="false" tabindex="-1">Plugins</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="#linkappearance" data-toggle="tab" role="tab" aria-selected="false" tabindex="-1">Appearance</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="#linkserver" data-toggle="tab" role="tab" aria-selected="false" tabindex="-1">Server</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="#linkreports" data-toggle="tab" role="tab" aria-selected="false" tabindex="-1">Reports</a>
                </li>
                <li class="nav-item" id="yui_3_17_2_1_1614770436101_55">
        <div class="popover-region collapsed" id="yui_3_17_2_1_1614770436101_54">
            <a href="#" class="nav-link d-inline-block popover-region-toggle position-relative" onclick="require(['local_edusupport/main'], function(MAIN){ MAIN.showBox(); }); return false;" style="padding-right:0; padding-left:0;" id="yui_3_17_2_1_1614770436101_53">
                <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" focusable="false" style=" -ms-transform: rotate(360deg); -webkit-transform: rotate(360deg); transform: rotate(360deg);" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24" id="yui_3_17_2_1_1614770436101_52"><path d="M21 12.22C21 6.73 16.74 3 12 3c-4.69 0-9 3.65-9 9.28c-.6.34-1 .98-1 1.72v2c0 1.1.9 2 2 2h1v-6.1c0-3.87 3.13-7 7-7s7 3.13 7 7V19h-8v2h8c1.1 0 2-.9 2-2v-1.22c.59-.31 1-.92 1-1.64v-2.3c0-.7-.41-1.31-1-1.62z" fill="#2E3942" opacity="0.8"></path><circle cx="9" cy="13" r="1" fill="#2E3942" opacity="0.8"></circle><circle cx="15" cy="13" r="1" fill="#2E3942" opacity="0.8"></circle><path d="M18 11.03A6.04 6.04 0 0 0 12.05 6c-3.03 0-6.29 2.51-6.03 6.45a8.075 8.075 0 0 0 4.86-5.89c1.31 2.63 4 4.44 7.12 4.47z" fill="#2E3942" opacity="0.8"></path></svg>
            </a>
        </div>
    </li><li class="nav-item" id="yui_3_17_2_1_1614770436101_47">
                    <a class="nav-link active" href="#linkdevelopment" data-toggle="tab" role="tab" aria-selected="true" tabindex="0" id="yui_3_17_2_1_1614770436101_46">Development</a>
                </li>
</ul>
```

**Comment by dasistwas** [comment]
This the top navbar, where it should actually appear:

```
<ul class="nav navbar-nav ml-auto align-items-center" id="yui_3_17_2_1_1614770436101_24">
            <li class="nav-item d-none d-lg-block">
                <div id="searchinput-navbar-603f7103ecb66603f7103cb4094" class="simplesearchform">
    <div class="collapse" id="searchform-navbar">
        <form autocomplete="off" action="https://sandbox.atingi.org/search/index.php" method="get" accept-charset="utf-8" class="mform form-inline searchform-navbar">
                <input type="hidden" name="context" value="1">
            <div class="input-group">
                <label for="searchinput-603f7103ecb66603f7103cb4094">
                    <span class="sr-only">Search</span>
                </label>
                    <input type="text" id="searchinput-603f7103ecb66603f7103cb4094" class="form-control withclear" placeholder="Search" aria-label="Search" name="q" data-region="input" autocomplete="off">
                    <a class="btn btn-close" data-action="closesearch" data-toggle="collapse" href="#searchform-navbar" role="button">
                        <i class="icon fa wp-close fa-fw " aria-hidden="true"></i>
                        <span class="sr-only">Close</span>
                    </a>
                <div class="input-group-append">
                    <button type="submit" class="btn btn-submit" data-action="submit">
                        <i class="icon fa wp-search fa-fw " aria-hidden="true"></i>
                        <span class="sr-only">Search</span>
                    </button>
                </div>
            </div>
        </form>
    </div>
    <a class="btn btn-open" data-toggle="collapse" data-action="opensearch" href="#searchform-navbar" role="button" aria-expanded="false" aria-controls="searchform-navbar">
        <i class="icon fa wp-search fa-fw " aria-hidden="true"></i>
        <span class="sr-only">Toggle search input</span>
    </a>
</div>
            </li>
            <li class="nav-item">
                </li><li class="dropdown nav-item languagemenu m-r-1 p-0">
    <a class="dropdown-toggle nav-link p-l-0 p-r-0" id="drop-down-603f7103ed165603f7103cb4097" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" href="#" title="Language" aria-controls="drop-down-menu-603f7103ed165603f7103cb4097">
        English
    </a>
    <div class="dropdown-menu" role="menu" id="drop-down-menu-603f7103ed165603f7103cb4097" aria-labelledby="drop-down-603f7103ed165603f7103cb4097">
                <a class="dropdown-item" role="menuitem" href="https://sandbox.atingi.org/admin/search.php?query&amp;lang=de_wp" title="Deutsch">Deutsch</a>
                <a class="dropdown-item" role="menuitem" href="https://sandbox.atingi.org/admin/search.php?query&amp;lang=en_wp" title="English">English</a>
                <a class="dropdown-item" role="menuitem" href="https://sandbox.atingi.org/admin/search.php?query&amp;lang=es_wp" title="Español">Español</a>
                <a class="dropdown-item" role="menuitem" href="https://sandbox.atingi.org/admin/search.php?query&amp;lang=fr_wp" title="Français">Français</a>
                <a class="dropdown-item" role="menuitem" href="https://sandbox.atingi.org/admin/search.php?query&amp;lang=pt_br_wp" title="Português">Português</a>
                <a class="dropdown-item" role="menuitem" href="https://sandbox.atingi.org/admin/search.php?query&amp;lang=ar_wp" title="بالعربي">بالعربي</a>
    </div>
</li>
            
            
            <li class="nav-item">
                <div class="popover-region collapsed popover-region-notifications" id="nav-notification-popover-container" data-userid="13305" data-region="popover-region">
    <div class="popover-region-toggle nav-link icon-no-margin" data-region="popover-region-toggle" role="button" aria-controls="popover-region-container-603f7103ed9d2603f7103cb4098" aria-haspopup="true" aria-label="Show notification window with no new notifications" tabindex="0">
                <i class="icon fa wp-bell fa-fw " title="Toggle notifications menu" aria-label="Toggle notifications menu"></i>
        <div class="count-container hidden" data-region="count-container" aria-label="There are 0 unread notifications">0</div>

    </div>
    <div id="popover-region-container-603f7103ed9d2603f7103cb4098" class="popover-region-container" data-region="popover-region-container" aria-expanded="false" aria-hidden="true" aria-label="Notification window" role="region">
        <div class="popover-region-header-container">
            <h3 class="popover-region-header-text" data-region="popover-region-header-text">Notifications</h3>
            <div class="popover-region-header-actions" data-region="popover-region-header-actions">        <a class="mark-all-read-button" href="#" title="Mark all as read" data-action="mark-all-read" role="button" aria-label="Mark all as read">
            <span class="normal-icon"><i class="icon fa fa-check fa-fw " aria-hidden="true"></i></span>
            <span class="loading-icon icon-no-margin"><i class="icon fa fa-circle-o-notch fa-spin fa-fw " title="Loading" aria-label="Loading"></i></span>
        </a>
        <a href="https://sandbox.atingi.org/message/notificationpreferences.php?userid=13305" title="Notification preferences" aria-label="Notification preferences">
            <i class="icon fa wp-cog fa-fw " aria-hidden="true"></i>
        </a>
</div>
        </div>
        <div class="popover-region-content-container" data-region="popover-region-content-container">
            <div class="popover-region-content" data-region="popover-region-content">
                        <div class="all-notifications" data-region="all-notifications" role="log" aria-busy="false" aria-atomic="false" aria-relevant="additions"></div>
        <div class="empty-message" tabindex="0" data-region="empty-message">You have no notifications</div>

            </div>
            <span class="loading-icon icon-no-margin"><i class="icon fa fa-circle-o-notch fa-spin fa-fw " title="Loading" aria-label="Loading"></i></span>
        </div>
                <a class="see-all-link" href="https://sandbox.atingi.org/message/output/popup/notifications.php">
                    <div class="popover-region-footer-container">
                        <div class="popover-region-seeall-text">See all</div>
                    </div>
                </a>
    </div>
</div>
            </li>
            <li class="nav-item">
                </li><li class="dropdown nav-item">
    <a class="nav-link dropdown-toggle nocaret" href="#" id="workplace-menulink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" aria-controls="workplace-menu-603f7103ef139603f7103cb4099">
    <i class="icon fa wp-launcher fa-fw " title="Toggle workplace launcher" aria-label="Toggle workplace launcher"></i>
    </a>
    <div id="workplace-menu-603f7103ef139603f7103cb4099" data-region="workplace-menu-content" class="workplace-menu p-0 dropdown-menu wp-boxshadow-4" aria-labelledby="workplace-menulink">
        <div class="d-flex flex-column">
            <div class="content card-deck-wrapper">
                <div class="card-deck d-flex flex-wrap columns-3 m-3">
                    <a class="card bg-white border-0 wpmenu-item my-2 d-flex flex-column text-center align-items-center justify-content-center m-0 p-0" href="https://sandbox.atingi.org/admin/tool/program/index.php">
                        <img class="icon" alt="" src="https://sandbox.atingi.org/theme/image.php/workplace_giz/theme/1614704380/menu/programs">
                        <span class="small mt-2 mb-2">Programs</span>
                    </a>
                    <a class="card bg-white border-0 wpmenu-item my-2 d-flex flex-column text-center align-items-center justify-content-center m-0 p-0" href="https://sandbox.atingi.org/admin/tool/certification/index.php">
                        <img class="icon" alt="" src="https://sandbox.atingi.org/theme/image.php/workplace_giz/theme/1614704380/menu/certifications">
                        <span class="small mt-2 mb-2">Certifications</span>
                    </a>
                    <a class="card bg-white border-0 wpmenu-item my-2 d-flex flex-column text-center align-items-center justify-content-center m-0 p-0" href="https://sandbox.atingi.org/admin/tool/tenant/users.php">
                        <img class="icon" alt="" src="https://sandbox.atingi.org/theme/image.php/workplace_giz/theme/1614704380/menu/tenant_management">
                        <span class="small mt-2 mb-2">Users</span>
                    </a>
                    <a class="card bg-white border-0 wpmenu-item my-2 d-flex flex-column text-center align-items-center justify-content-center m-0 p-0" href="https://sandbox.atingi.org/admin/tool/tenant/details.php">
                        <img class="icon" alt="" src="https://sandbox.atingi.org/theme/image.php/workplace_giz/theme/1614704380/menu/tenant_appearance">
                        <span class="small mt-2 mb-2">Appearance</span>
                    </a>
                    <a class="card bg-white border-0 wpmenu-item my-2 d-flex flex-column text-center align-items-center justify-content-center m-0 p-0" href="https://sandbox.atingi.org/admin/tool/organisation/index.php">
                        <img class="icon" alt="" src="https://sandbox.atingi.org/theme/image.php/workplace_giz/theme/1614704380/menu/organization_structure">
                        <span class="small mt-2 mb-2">Organisation structure</span>
                    </a>
                    <a class="card bg-white border-0 wpmenu-item my-2 d-flex flex-column text-center align-items-center justify-content-center m-0 p-0" href="https://sandbox.atingi.org/admin/tool/dynamicrule/index.php">
                        <img class="icon" alt="" src="https://sandbox.atingi.org/theme/image.php/workplace_giz/theme/1614704380/menu/dynamic_rules">
                        <span class="small mt-2 mb-2">Dynamic rules</span>
                    </a>
                    <a class="card bg-white border-0 wpmenu-item my-2 d-flex flex-column text-center align-items-center justify-content-center m-0 p-0" href="https://sandbox.atingi.org/admin/tool/reportbuilder/index.php">
                        <img class="icon" alt="" src="https://sandbox.atingi.org/theme/image.php/workplace_giz/theme/1614704380/menu/report_builder">
                        <span class="small mt-2 mb-2">Report builder</span>
                    </a>
                </div>
                <div class="border-top">
                    <div class="mx-4 mt-4 font-weight-bold small">
                        Global
                    </div>
                    <div class="card-deck d-flex flex-wrap columns-3 mx-3 mb-3">
                            <a class="card bg-white border-0 wpmenu-item my-3 d-flex flex-column text-center align-items-center justify-content-center m-0 p-0" href="https://sandbox.atingi.org/course/management.php">
                                <img class="icon" alt="" src="https://sandbox.atingi.org/theme/image.php/workplace_giz/theme/1614704380/menu/courses">
                                <span class="small mt-2 mb-2">Courses</span>
                            </a>
                            <a class="card bg-white border-0 wpmenu-item my-3 d-flex flex-column text-center align-items-center justify-content-center m-0 p-0" href="https://sandbox.atingi.org/admin/tool/certificate/manage_templates.php">
                                <img class="icon" alt="" src="https://sandbox.atingi.org/theme/image.php/workplace_giz/theme/1614704380/menu/certificates">
                                <span class="small mt-2 mb-2">Certificates</span>
                            </a>
                            <a class="card bg-white border-0 wpmenu-item my-3 d-flex flex-column text-center align-items-center justify-content-center m-0 p-0" href="https://sandbox.atingi.org/admin/tool/tenant/allusers.php">
                                <img class="icon" alt="" src="https://sandbox.atingi.org/theme/image.php/workplace_giz/theme/1614704380/menu/users">
                                <span class="small mt-2 mb-2">All users</span>
                            </a>
                            <a class="card bg-white border-0 wpmenu-item my-3 d-flex flex-column text-center align-items-center justify-content-center m-0 p-0" href="https://sandbox.atingi.org/admin/tool/tenant/index.php">
                                <img class="icon" alt="" src="https://sandbox.atingi.org/theme/image.php/workplace_giz/theme/1614704380/menu/tenants">
                                <span class="small mt-2 mb-2">All tenants</span>
                            </a>
                            <a class="card bg-white border-0 wpmenu-item my-3 d-flex flex-column text-center align-items-center justify-content-center m-0 p-0" href="https://sandbox.atingi.org/admin/tool/wp/exportimport.php">
                                <img class="icon" alt="" src="https://sandbox.atingi.org/theme/image.php/workplace_giz/theme/1614704380/menu/exportimport">
                                <span class="small mt-2 mb-2">Migration</span>
                            </a>
                    </div>
                </div>
            </div>
        </div>
    </div>
</li>
            
            <li class="nav-item" id="yui_3_17_2_1_1614770436101_23">
        <div class="popover-region collapsed" id="yui_3_17_2_1_1614770436101_22">
            <a href="#" class="nav-link d-inline-block popover-region-toggle position-relative" onclick="require(['local_edusupport/main'], function(MAIN){ MAIN.showBox(); }); return false;" style="padding-right:0; padding-left:0;" id="yui_3_17_2_1_1614770436101_21">
                <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" focusable="false" style=" -ms-transform: rotate(360deg); -webkit-transform: rotate(360deg); transform: rotate(360deg);" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24" id="yui_3_17_2_1_1614770436101_20"><path d="M21 12.22C21 6.73 16.74 3 12 3c-4.69 0-9 3.65-9 9.28c-.6.34-1 .98-1 1.72v2c0 1.1.9 2 2 2h1v-6.1c0-3.87 3.13-7 7-7s7 3.13 7 7V19h-8v2h8c1.1 0 2-.9 2-2v-1.22c.59-.31 1-.92 1-1.64v-2.3c0-.7-.41-1.31-1-1.62z" fill="#2E3942" opacity="0.8"></path><circle cx="9" cy="13" r="1" fill="#2E3942" opacity="0.8"></circle><circle cx="15" cy="13" r="1" fill="#2E3942" opacity="0.8"></circle><path d="M18 11.03A6.04 6.04 0 0 0 12.05 6c-3.03 0-6.29 2.51-6.03 6.45a8.075 8.075 0 0 0 4.86-5.89c1.31 2.63 4 4.44 7.12 4.47z" fill="#2E3942" opacity="0.8"></path></svg>
            </a>
        </div>
    </li><li class="nav-item d-flex align-items-center">
                <div id="usermenu-603f7103f2292603f7103cb40910" class="dropdown usermenu" data-region="usermenu" role="navigation">

    <a class="btn dropdown-toggle" href="#" role="button" id="usermenulink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
            <span class="userbutton notloggedinas">
                <img src="https://sandbox.atingi.org/theme/image.php/workplace_giz/core/1614704380/u/f1" class="userpicture defaultuserpic" width="50" height="50" aria-hidden="true">
            </span>
    </a>
    <div data-region="usermenu-content" class="workplace-usermenu p-0 dropdown-menu wp-boxshadow-4" aria-labelledby="usermenulink">
        <div class="userinfo p-4 d-flex align-items-center border-bottom">
            <img src="https://sandbox.atingi.org/theme/image.php/workplace_giz/core/1614704380/u/f1" class="userpicture defaultuserpic" width="50" height="50" aria-hidden="true">
                <div class="userdetails">
                    <div class="username">
                        <strong>David Bogner</strong>
                    </div>
                    <div class="useremail small">
                        david.bogner@gopa.de
                    </div>
                </div>
            <div class="navitems ml-auto">
                     <a href="https://sandbox.atingi.org/user/profile.php?id=13305" class="useraction" data-toggle="tooltip" data-placement="top" title="Profile">
                        <i class="icon fa wp-user fa-fw " aria-hidden="true"></i>
                     </a>
                     <a href="https://sandbox.atingi.org/course/switchrole.php?id=1&amp;switchrole=-1&amp;returnurl=%2Fadmin%2Fsearch.php%3Fquery" class="useraction" data-toggle="tooltip" data-placement="top" title="Switch role to...">
                        <i class="icon fa wp-switch-roles fa-fw " aria-hidden="true"></i>
                     </a>
                     <a href="https://sandbox.atingi.org/login/logout.php?sesskey=ALUU4NIHfi" class="useraction" data-toggle="tooltip" data-placement="top" title="Log out">
                        <i class="icon fa wp-logout fa-fw " aria-hidden="true"></i>
                     </a>
            </div>
        </div>
        <div class="h-100 courses-container" style="overflow-y: auto; overflow-x:hidden;">
            <div class="list-group">
                <h3 class="popover-region-header-text px-3 py-2">Recently accessed courses</h3>
                <a href="https://sandbox.atingi.org/course/view.php?id=526" class="list-group-item course-listitem d-flex border-0 p-3">
                    <div class="course-list-img rounded mr-2" style="background-image: url(&quot;https://sandbox.atingi.org/pluginfile.php/62530/course/overviewfiles/hand-1917895_640.png&quot;);">
                        <span class="sr-only">Course image</span>
                    </div>
                    <div class="coursename">
                    <span class="sr-only">
                        Course name
                    </span>
                    Support
                    </div>
                </a>
                <a href="https://sandbox.atingi.org/course/view.php?id=63" class="list-group-item course-listitem d-flex border-0 p-3">
                    <div class="course-list-img rounded mr-2" style="background-image: url(&quot;https://sandbox.atingi.org/pluginfile.php/620/course/overviewfiles/127459-4cc327d5-9362-40cf-9246-03384c621e50.png&quot;);">
                        <span class="sr-only">Course image</span>
                    </div>
                    <div class="coursename">
                    <span class="sr-only">
                        Course name
                    </span>
                    atingi Digital Learning Workshop for Content and Course Development Rwanda
                        <div class="courseduedate font-mini text-uppercase mt-1">
                            Ends: 1/12/21
                        </div>
                    </div>
                        <div class="ml-auto align-self-center d-flex align-items-center">
                            <div class="progress mr-2">
                                <div class="progress-bar bg-info bar" role="progressbar" aria-valuenow="0" style="width: 0%" aria-valuemin="0" aria-valuemax="100"></div>
                            </div>
                            <div class="small text-nowrap progress-text">
                                0 %
                            </div>
                        </div>
                </a>
            </div>
        </div>
    </div>
</div>
            </li>
        </ul>
```

**Comment by dasistwas** [comment]
@winnidat aria-label="Site navigation" is not a valid css selector. It changes with the language.

```
<nav class="fixed-top d-print-none navbar navbar-dark bg-brand navbar-expand moodle-has-zindex d-flex wp-boxshadow-3  pl-0  pr-2 " aria-label="Navigation du site">
```
Workaround on the sandbox:
```

.nav-tabs .popover-region {
    display: none;
}
```
Maybe that is good:

```

$(result).insertBefore($('nav.fixed-top .nav .nav-item:last-child'));
```

**Comment by dasistwas** [comment]
@winnidat git pull before changes: integrated fixes from upstream

**Comment by dasistwas** [comment]
Fixed with eb68d40abcd64618f1736a7ca2a151afd05d31b4
